CRM/Contribute/DAO/ContributionRecur.php

```diff
@@ -587,7 +591,7 @@ public static function &fields() {
           'import' => TRUE,
           'where' => 'civicrm_contribution_recur.contribution_status_id',
           'export' => TRUE,
-          'default' => '1',
+          'default' => '2',
```

Comment:
Might be nice to have comment 
// ie Pending

Reply:
Agree it would be nice, but we can't put comments in a generated DAO file.